sensor_msgs/PointCloud2 input_cloud
---
sensor_msgs/PointCloud2 segmented_cloud
